### PR TITLE
VEC-140 Add helm charts for quote-search Proximus app

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: aerospike-proximus-examples/quote-semantic-search/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+              name: data
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: PROXIMUS_HOST
+              value: "as-quote-search-aerospike-proximus.aerospike.svc.cluster.local"
```
Comment (arrowplum) [inline aerospike-proximus-examples/quote-semantic-search/templates/deployment.yaml:53]: I think this is currently hardcoded, right? Shouldn't it come from env?

**Comment by arrowplum** [comment]
Otherwise was able to get it up and running no problem at least on gke. had to change the deployment.yaml for the host and use the official docker image. otherwise :man_cook:

File: aerospike-proximus-examples/quote-semantic-search/values.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Default values for quote-semantic-search.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: davi17g/quote-search
```
Comment (arrowplum) [inline aerospike-proximus-examples/quote-semantic-search/values.yaml:8]:
```suggestion
  repository: aerospike/quote-search-example
```

File: aerospike-proximus-examples/quote-semantic-search/templates/deployment.yaml (hunk 1 above)
Comment (davi17g) [reply aerospike-proximus-examples/quote-semantic-search/templates/deployment.yaml:53]: yes, I'll expose all parameters to `values.yaml `file

**Review by arrowplum — COMMENTED** [review]
LG2M

I at least want us to get the repo stuff merged (the docs)

**Review by arrowplum — APPROVED** [review]

**Pull request merged**